posydon/binary_evol/flow_chart.py

```diff
@@ -293,6 +293,11 @@
                 POSYDON_FLOW_CHART[(s1, s2, b, e)] = 'step_end'
                 POSYDON_FLOW_CHART[(s2, s1, b, e)] = 'step_end'
 
+for b in BINARY_STATES_ALL:
+    for s in ['massless_remnant']:
+        for e in BINARY_EVENTS_ALL:
+            POSYDON_FLOW_CHART[(s, s, b, e)] = 'step_end'
+
```

Comment:
Wouldn't it be simpler to include this in the previous set of loops as
`POSYDON_FLOW_CHART[(s2, s2, b, e)] = 'step_end'`?
Maybe we'd need to add
`if (s1==STAR_STATES_CO[0]):` before it in case, the flow chart don't like to overwrite stuff.

Reply:
I would say that this provides a clearer view of what is being added to the `FLOW_CHART` for the user.

Isn't this code only ran when the steps are loaded? So the overhead shouldn't be that much.

Reply:
OK, could you please add a comment, what is added here, when you argue with being better readable for the user.